maquetado/setup/scss/titles.scss

```diff
@@ -0,0 +1,20 @@
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+
+.title {
+  display: flex;
+  position: relative;
+
+  .title-decoration{
```

Comment:
Before you go on, I know in style guides it says you should nest styles, try not to do so. It makes it harder to override